app/tests/src/ingestion/test_pdf_postprocess.py

```diff
@@ -86,41 +86,55 @@ def test_concatenate_list_items():
             text="Introduction.",
             type=TextType.NARRATIVE_TEXT,
             headings=[Heading(title="Overview", level=1)],
+            page_number=1,
         ),
         EnrichedText(
             text="First item.",
             type=TextType.LIST_ITEM,
             headings=[Heading(title="Overview", level=1)],
+            page_number=1,
         ),
         EnrichedText(
             text="Second item.",
             type=TextType.LIST_ITEM,
             headings=[Heading(title="Overview", level=1)],
+            page_number=1,
         ),
         EnrichedText(
             text="Another narrative text.",
             type=TextType.NARRATIVE_TEXT,
             headings=[Heading(title="Overview", level=1)],
+            page_number=1,
         ),
         EnrichedText(
             text="Narrative starting a new list: ",  # ending space is intentional
             type=TextType.NARRATIVE_TEXT,
             headings=[Heading(title="Overview", level=1)],
+            page_number=1,
         ),
         EnrichedText(
             text="First item in new list.",
             type=TextType.LIST_ITEM,
             headings=[Heading(title="Overview", level=1)],
+            page_number=1,
         ),
         EnrichedText(
             text="Second item in new list.",
             type=TextType.LIST_ITEM,
             headings=[Heading(title="Overview", level=1)],
+            page_number=1,
         ),
         EnrichedText(
-            text="New list item in new section.",
+            text="New list item in new section",
             type=TextType.LIST_ITEM,
             headings=[Heading(title="Section 1", level=1)],
+            page_number=1,
+        ),
+        EnrichedText(
+            text="with continuing sentence on next page",
+            type=TextType.NARRATIVE_TEXT,
```

Comment:
I don't know if unstructured would categorize a list item that overflows onto the next page as `NARRATIVE_TEXT` or `LIST_ITEM`.
If it was `LIST_ITEM`, then `_group_list_texts()` would make it a new item, which is not the correct semantic.

Reply:
My wild guess is that it'll be NarrativeText, with about 50% confidence 😂  I'd say let's try it and see what happens!